compiler/il/VectorOperations.enum

```diff
@@ -739,6 +739,73 @@ VECTOR_OPERATION_MACRO(\
    /* .description             =    get vector element, returns scalar */ \
 )
 //
+// masked one-type operations
+//
```

Comment:
Can you add the other opcodes in this PR, such as vand, vor, vxor, vmin, vmax, vabs, sqrt... Might as well save the added trouble later.

Reply:
done